net/dnsviz/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# $Id$
+
+PortSystem         1.0
+
+PortGroup          github 1.0
+PortGroup          python 1.0
+
+github.setup       dnsviz dnsviz 0.6.5 v
+github.tarball_from releases
+
+categories         net
+platforms          darwin
+license            GPL-2+
+supported_archs    noarch
+maintainers        deccio.net:casey
```

Comment:
You can add `openmaintainer`.

See https://guide.macports.org/#project.update-policies.nonmaintainer